projects/packages/search/.gitattributes

```diff
@@ -19,5 +21,7 @@ package.json                                             export-ignore
 /src/**/test/**                                          production-exclude
 /src/**/*.scss                                           production-exclude
 # JS scripts to integrate with customizer and wigets might need to be explicitly included.
-/src/**/*.[jt]s?(x)                                      production-exclude
+/src/**/*.js                                             production-exclude
+/src/**/*.jsx                                            production-exclude
+/src/**/*.ts                                             production-exclude
```

Comment:
I've switched to more explicit declarations here because the pattern does not seem to work:
https://github.com/Automattic/jetpack-search/tree/e408700a61a7937263cdb48320acc91def69a95d/src/customberg

Reply:
The `?(x)` bit isn't a supported part of the syntax.